File: src/models/Passenger.java
package models;

public class Passenger {
	private String firstName;
	private String lastName;
	private Ticket ticket = null;
	
	public Passenger(String firstName, String lastName) {
		super();
		this.firstName = firstName;
		this.lastName = lastName;
	}

	@Override
	public String toString() {
		return "Passenger [firstName=" + firstName + ", lastName=" + lastName
				+ ", ticket=" + ticket + "]";
	}
	
	
}

File: AirlineServices/AirlineServices/Models/Passenger.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Web;

namespace AirlineServices.Models
{
    public class Passenger : Person 
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int id { get; set; }
        [ForeignKey("PassengerId")]
        public virtual ICollection<Ticket> tickets { get; set; }

        private DateTime createDate = DateTime.Now;

        public DateTime CreateDate { get { return createDate; } set { lastModified = createDate; } }


        private DateTime lastModified = DateTime.Now;

        public DateTime LastModified { get { return lastModified; } set { lastModified = DateTime.Now; } }

        public bool hasActiveTickets()
        {
            var activeTickets = tickets.Where(s => s.status == TicketStatusType.BOOKED || s.status == TicketStatusType.CONFIRMED).ToList();
            return (activeTickets.Count > 0);
        }
    }
}